src/main/java/org/sonatype/nexus/plugins/conda/internal/proxy/CondaProxyFacetImpl.java

```diff
@@ -117,31 +117,31 @@ protected Content store(final Context context, final Content content) throws IOE
       case CHANNEL_INDEX_HTML:
         return putMetadata(content,
             assetKind,
-            condaPathUtils.buildAssetPath(matcherState, CondaPathUtils.INDEX_HTML));
+            condaPathUtils.buildAssetPath(matcherState, INDEX_HTML));
       case CHANNEL_DATA_JSON:
         return putMetadata(content,
             assetKind,
-            condaPathUtils.buildAssetPath(matcherState, CondaPathUtils.CHANNELDATA_JSON));
+            condaPathUtils.buildAssetPath(matcherState, CHANNELDATA_JSON));
       case CHANNEL_RSS_XML:
         return putMetadata(content,
             assetKind,
-            condaPathUtils.buildAssetPath(matcherState, CondaPathUtils.RSS_XML));
+            condaPathUtils.buildAssetPath(matcherState, RSS_XML));
       case ARCH_INDEX_HTML:
         return putMetadata(content,
             assetKind,
-            condaPathUtils.buildArchAssetPath(matcherState, "/index.html"));
+            condaPathUtils.buildArchAssetPath(matcherState, INDEX_HTML));
       case ARCH_REPODATA_JSON:
         return putMetadata(content,
             assetKind,
-            condaPathUtils.buildArchAssetPath(matcherState, "/repodata.json"));
+            condaPathUtils.buildArchAssetPath(matcherState, REPODATA_JSON));
       case ARCH_REPODATA_JSON_BZ2:
         return putMetadata(content,
             assetKind,
-            condaPathUtils.buildArchAssetPath(matcherState, "/repodata.json.bz2"));
+            condaPathUtils.buildArchAssetPath(matcherState, REPODATA_JSON_BZ2));
       case ARCH_REPODATA2_JSON:
         return putMetadata(content,
             assetKind,
-            condaPathUtils.buildArchAssetPath(matcherState, "/repodata2.json"));
+            condaPathUtils.buildArchAssetPath(matcherState, REPODATA2_JSON));
```

Comment:
I looked at how this would have gotten assembled, and it was definitely wrong (you'd end up with `//repodata2.json`) so I updated these to what should be correct

Reply:
Noice!